src/components/cell-control/input.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import CellControl from '../cell-control/cell-control.jsx';
+import InputControl from '../control/input.jsx';
+
+function Input(props) {
+  return (
+    <CellControl
+      labelledBy={props.labelledBy}
+      readOnly={props.readOnly}
+    >
+      <InputControl
```

Comment:
I don't hook all the props because I want to figure out what we need for custom forms. And how to implement "test coverage": do I repeat the tests? do I only test the InputControl? etc etc. I think I will have a better idea on how to do this with a few more cell controls.